PolishWithBlondes/src/components/hero/hero.tsx

```diff
@@ -0,0 +1,30 @@
+import heroPic1 from '../../assets/hero-pic1.png';
+import './hero.scss';
+
+export const Hero = () => {
+  return (
+    <div className="hero">
+      <div className="col left">
+        <h1 className="hero-title">
+          <strong>Cześć!</strong>
+        </h1>
+        <h3 className="hero-sub">
+          We're Kasia, Iza and Zuzia and together we form{' '}
+          <a href="https://www.youtube.com/@polishwithblondes" target="_blank">
```

Comment:
The link color looks really simillar to the rest. Maybe it could stand out a bit more